keyboards/kumo/keymaps/LEdiodes/rules.mk

```diff
@@ -0,0 +1,70 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
All of this stuff isn't needed.

Just the stuff in needed to be changed from default are needed here.